### ISSUE TITLE
Unlimited issue

### ISSUE THREAD

**Issue by rastermechanism** [inokawa/remark-docx#55]
**Describe the bug**
It has many symbol and parsing issue

**To Reproduce**
Fails on most of the math formula

**Expected behavior**
It should render correctly

**Additional context**
I have tried with many issue and even corrected it. With docxjs and parsing issue. But seems I have open pandora box. More I correct, more I get. So finally I have decided to give up and try something else, planning to use natively supported api. 

Correction I have made 
with Docxjs -- implemented code block 
Math: Symbol Nebula 
Still issue when $ is written in terms of monetry valaue, I know even markdown parser fails to resolve it. 
However this was not the only issue your code fails to resolve many issues like Big bracket like ( symbol in permutation. 
I hope someone has time to resolve all this issue before we can call it a production package . 

Here is demo for improvements   [Demo Link](https://www.assignmenthelp.net/massivemark)

**Comment by mayank1513** [comment]
I have created this PR - https://github.com/inokawa/remark-docx/pull/79 

Meanwhile, you can try out [`@m2d/remark-docx`](https://npmjs.com/package/@m2d/remark-docx)

**Comment by inokawa** [comment]
Duplicate of #4
